### PR TITLE
AT fix

### PR DESCRIPTION
**Pull request by mloubout**
Profiler switched on at declaration to have proper number outputed. Reduction clause are ignored if the profiler is turned on inside `at_controller` afterward.

### CODE REVIEW

File: examples/acoustic_example.py [hunk 1]
```diff
@@ -3,15 +3,16 @@
...
 model = IGrid()
 model0 = IGrid()
 model1 = IGrid()
 model.shape = dimensions
 model0.shape = dimensions
 model1.shape = dimensions
-origin = (0., 0.)
-spacing = (20., 20.)
+origin = (0., 0., 0.)
+spacing = (20., 20., 20.)
```
Comment (FabioLuporini) [inline examples/acoustic_example.py]: This is fixed by #115 so we should first merge that, and then rebase this on top of it

**Comment by FabioLuporini** [comment]
Apart from the minor comment, this should be merged ASAP

File: examples/Acoustic_codegen.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ def damp_boundary(damp):
         if auto_tune:  # auto tuning with dummy forward operator
             fw = ForwardOperator(self.model, self.src, self.damp, self.data,
                                  time_order=self.t_order, spc_order=self.s_order,
-                                 save=False)
+                                 save=False, profile=False)
```
Comment (vincepandolfo) [inline examples/Acoustic_codegen.py]: `profile` defaults to `False`. Why is this needed?

Comment (mloubout) [inline examples/Acoustic_codegen.py]: Should be True my bad.

**Comment by mloubout** [comment]
This one is good to go I think

**Comment by mlange05** [comment]
Yes, it is. Merging...
